mlkem/randombytes.h

```diff
@@ -9,11 +9,21 @@
 #include <stdint.h>
 
 #include "cbmc.h"
+#include "common.h"
 
-void randombytes(uint8_t *out, size_t outlen)
+
+#if defined(MLK_CUSTOM_RANDOMBYTES)
+static MLK_INLINE void mlk_randombytes(uint8_t *out, size_t outlen)
 __contract__(
   requires(memory_no_alias(out, outlen))
-  assigns(memory_slice(out, outlen))
-);
+  assigns(memory_slice(out, outlen)));
+#else
+
+void randombytes(uint8_t *out, size_t outlen);
```

Comment:
Don't you need a CBMC spec in this case, too?

Reply:
yes - stupid me. fixed.